test/make.sh

```diff
@@ -143,16 +132,15 @@ function check_trailing_whitespace() {
 
 function generate_docs() {
```

Comment:
This version of the function matches [module template](https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/test/make.sh#L127-L142) but doesn't match [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/test/make.sh#L144-L162). Which is the latest, @nick4fake?